### DISCUSSION TITLE
invalid slug naming cause unknow error

### DISCUSSION THREAD

**Discussion by mcimam** [payloadcms/payload#2138] [Feature Requests & Ideas]
# Bug Report

If slug contain space in it The API will produce 404 error. In Admin panel it will only produce 'unknow error occured'. This is really confusing plus in documentation it just said that slug must have  'URL-friendly string '.

## Steps to Reproduce

1. Add space in slug collection (ex: sale orders)
2. Open related collection in admin panel
3. See console for 404 error

## Other Details
payload : 1.6.11

I think, adding validation to check naming will help. Adding trim function or replace space with - will also helped.

**Comment by jmikrut** [comment]
Hey @mcimam — this is actually intended behavior. We should indeed throw an error on startup on if the slug is not URL-friendly, but that will be a future enhancement. For now, I will convert this to a discussion and tag it as a feature request. We will get to this for sure.

Thanks for bringing this up!
